Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -4191,6 +4191,11 @@ RefPtr<CSSValue> ComputedStyleExtractor::valueForPropertyInStyle(const RenderSty
         return createConvertingToCSSValueID(style.scrollSnapStop());
     case CSSPropertyScrollSnapType:
         return valueForScrollSnapType(style.scrollSnapType());
+    case CSSPropertyScrollbarColor:
+        // TODO correctly handle inheritance
+        if (!style.scrollbarColor())
+            return CSSPrimitiveValue::create(CSSValueAuto);
```

Comment:
Need to work out how to correctly handle the case where this is something like unset where instead of auto it should inherit. I believe the test failure is as a result of this line of code saying theres no value so it must be auto (which is correct) but unset should be handled. Now this should be handled I'm not yet sure.

Perhaps std::optional isn't the best data type? The equivalent is used inside of Chrome and that inherits fine though.

Reply:
I think the issue is in CSSProperties.json, this code is fine